juri/styles.css

```diff
@@ -0,0 +1,179 @@
+html{
+  box-sizing: border-box;
+}
+*, *::before, *::after {
+  box-sizing: inherit;
+  box-sizing: border-box;
+}
+
+html{
+  position: relative;
+}
+
+:root{
+  --background-color:#F8F8F8;
+  --h1-color: #515CC6;
+  --h2-color: #575767;
+  --hr-color:#D0D0D0;
+  --top-color:#575767;
+
+
+  --font-family: 'Inter', sans-serif;
+  --main-font-family : var(--font-family);
+
+
+
+ /*
+  --font-size-p: 14px;
+  --font-size-h1: 32px;
+  --font-size-main: 18px;
+  --font-color-h1: black ;
+  --color-p-label: #B9B9BE;
+  --font-weight-h1-h2: 700;
+  --font-weight-p: 600;
+  --font-weight-li: 500;
+  --subtitle-color: #575767; */
+
+}
+
+
+body {
+  background-color: var(--background-color);
+}
+
+h1 {
+  
+  font-family: var(--main-font-family);
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  color: var(--h1-color);
+}
+h2 {
+  font-family: var(--main-font-family);
+  font-style: bold;
+  font-size: 18px;
+  line-height: 18px;
+  color:var(--h2-color);
+
+}
+hr {
+color: var(--hr-color); 
+;
+}
+.top {
```

Comment:
You forgot to remove left padding of the list, check please the design: only `<p>` elements have padding left.